### DISCUSSION TITLE
Default caching strategy

### DISCUSSION THREAD

**Discussion by stephzero1** [verdaccio/verdaccio#4238] [Q&A] [answered]
In my experience is not true that installing Verdaccio (as docker container) set the default caching strategy as true.

In my case (stock docker installation) I need to edit the config.yaml by myself, and add the flag cache:true:
![image](https://github.com/verdaccio/verdaccio/assets/25548571/5c6b1f05-b3a1-4455-a712-7f8010ba59f7)

I think the doc should say it, instead to say that by default is enabled:
![image](https://github.com/verdaccio/verdaccio/assets/25548571/ddd064fe-67f3-4372-a10a-08529f18b33a)

https://verdaccio.org/docs/caching

**Comment by juanpicado** [comment]
That might be a bug in the latest versions and would worth verify when actually stopped to work

**Comment by juanpicado** [comment, marked as answer]
~Just use `v5.25.0` or us the `cache: true` as workaround in the meantime is fixed.~ works for me
